lib/webpack/markdownLoader.js

```diff
@@ -5,26 +5,35 @@ const { EventEmitter } = require('events')
 const { getOptions } = require('loader-utils')
 const { inferTitle, extractHeaders, parseFrontmatter } = require('../util')
 const LRU = require('lru-cache')
+const { mdExplodeIncludes } = require('./util')
 
 const cache = LRU({ max: 1000 })
 const devCache = LRU({ max: 1000 })
 
-module.exports = function (src) {
+module.exports = async function (src) {
+  const cb = this.async()
+  try {
+  /* IMPORTANT: I didn't indent these lines to hopefully get a better looking diff */
```

Comment:
I have un-indented the lines in the `try { ... } catch` block to show a better looking diff. Please re-indent when PR is accepted. :blush: